src/sensor.cpp

```diff
@@ -509,10 +507,9 @@ void log_callback_end( uint32_t fps,
                     diff = environment::get_instance().get_time_service()->get_time() - system_time;
                     if (diff >10 )
                         LOG_DEBUG("!! Frame memcpy took " << diff << " msec");
-                    if (!requires_processing)
```

Comment:
Looks like `requires_processing` is no longer in use -- please remove (line 417)

Reply:
done